src/NexusMods.Abstractions.Collections/NexusCollectionLoadoutGroup.cs

```diff
@@ -39,4 +43,88 @@ public partial struct ReadOnly
                 .Where(static model => model.IsValid());
         }
     }
+
+    public static async Task<EntityId> MakeEditableLocalCollection(IConnection conn, EntityId collId, string newName)
+    {
+        var cloneId = await CollectionGroup.Clone(conn, collId);
+        var cloneEnt = Load(conn.Db, cloneId);
+                
+        using var tx = conn.BeginTransaction();
+        // Remap the name
+        tx.Add(cloneId, LoadoutItem.Name, newName);
+        // Make it editable
+        tx.Add(cloneId, CollectionGroup.IsReadOnly, false);
+        // Retract the Nexus references as this is no longer associated with the official collection
+        tx.Retract(cloneId, RevisionId, RevisionId.Get(cloneEnt));
+        tx.Retract(cloneId, CollectionId, CollectionId.Get(cloneEnt));
+        tx.Retract(cloneId, LibraryFileId, LibraryFileId.Get(cloneEnt));
+
+        // Retract the Nexus references in items so that this is no longer associated with the official collection
+        var db = conn.Db;
+        foreach (var item in conn.Query<EntityId>("SELECT Id FROM mdb_NexusCollectionItemLoadoutGroup(Db=>$Db) WHERE Parent = $Collection AND IsRequired = True", new { Db = db, Collection = cloneId }))
```

Comment:
Why the `AND IsRequred = True` check?

Isn't that just going to miss optional mods that also need to be stripped of attributes?
For example Bundled Optional mods?